components/raceDetails.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
-import { FlatList, StyleSheet, View } from 'react-native';
-import { Header, List, ListItem, Text } from 'react-native-elements';
+import { View, SectionList } from 'react-native';
+import { List, ListItem, Text } from 'react-native-elements';
 
 export default class RaceList extends React.Component {
     state = {
         race: {},
         refreshing: false
-    }
+    };
+
+    categories = ['zene', 'u16', '16-20', '20-30', '30-40', '40-50', '50+'];
 
     componentDidMount() {
         this.loadRace()
     }
 
+    getSectionsWithData = () => {
+        const sections = [];
+        if(this.state.race.race_results) {
```

Comment:
Destructuring this above would make it much more readable.

Reply:
👍